Neos.Error.Messages/Classes/Message.php

```diff
@@ -102,10 +111,19 @@ public function getArguments(): array
     }
 
     /**
-     * @return string
+     * @return bool
+     * @api
+     */
+    public function hasTitle(): bool
+    {
+        return $this->title !== null && $this->title !== '';
+    }
+
+    /**
+     * @return string|null
      * @api
      */
-    public function getTitle(): string
+    public function getTitle()
```

Comment:
Nope, since it will fail if a null comes in (see neos/neos-development-collection#1666)